tests/Inspetor/Model/InspetorSaleTest.php

```diff
@@ -90,15 +90,15 @@ public function testIfIsNotValidWhenStatusIsNotValidIsNull() {
         $sale->isValid();
     }
 
-    public function testIfIsNotValidWhenIsFraudIsNull() {
-        $sale = $this->getDefaultSale();
-        $sale->setIsFraud(null);
+    // public function testIfIsNotValidWhenIsFraudIsNull() {
```

Comment:
🤔

Reply:
We should change this to test the isValidUpdate

Reply:
Someone (theo) forgot that.